src/routes/Cart.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import CartCardAndTotalBox from "../components/cart/CartCardAndTotalBox";
+import { Navbar } from "../components/navbar/Navbar";
+
+const Cart = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
no need to render navbar in all routes instead you can keep it outside of routes so it will renders in all route